charts/consul/values.yaml

```diff
@@ -2753,6 +2753,27 @@ meshGateway:
     # DNS entry to point to your mesh gateways.
     static: ""
 
+  # This configures the [`PodDisruptionBudget`](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
```

Comment:
May be I am missing something, but I don't see a `disruptionBudget` stanza for `apiGateways` but it seems to be referenced in the helm template files. Is this intentional?

Reply:
please check line #3276, this is for consul api gateway, connect inject uses CRD gateway as per my understanding, it has its own default pdb yaml.

Reply:
Ahh my bad 🤦 . The diff confused me. Thanks for pointing out